prql-compiler/src/prql.pest

```diff
@@ -25,7 +25,7 @@ type_term = { ident ~ type_def? }
 
 table = { "table" ~ ident ~ "=" ~ nested_pipeline }
 
-pipe = _{ NEWLINE | "|" }
+pipe = _{ "|" | NEWLINE* }
```

Comment:
I _think_ this should be a `+` rather than a `*` — i.e. it should have at least one newline. I'm kinda surprised the whole thing works with a `*` — meaning nothing at all is a valid `pipe`!

```suggestion
pipe = _{ "|" | NEWLINE+ }
```

Is that right?

Reply:
oops, you're right. I've updated my PR!